parser/tests/integration/test_parser_integration.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import pathlib
+from openfoodfacts_taxonomy_parser import parser
+
+# taxonomy in text format : test.txt
+TEST_TAXONOMY_TXT = str(pathlib.Path(__file__).parent.parent / "data" / "test")
+
+
+@pytest.fixture
+def new_session():
+    x = parser.Parser()
+    # delete all the nodes and relations in the database
+    query="MATCH (n) DETACH DELETE n"
+    x.session.run(query)
+    return x
```

Comment:
it's strange to name it new_session and that it returns a Parser instance !

I would have written:

```suggestion
@pytest.fixture(auto_use=True)
def test_setup():
    # delete all the nodes and relations in the database
    query="MATCH (n) DETACH DELETE n"
    parser.Parser().session.run(query)
```
With autouse, you don't need to add it to tests.

It's ok to create a second Parser in your test, this is not that high a cost and it's more explicit.